src/routes/game/components/GameMenu.vue

```diff
@@ -17,19 +17,19 @@
       <!-- Menu -->
       <v-list id="game-menu" class="text-surface-1" bg-color="surface-2">
         <v-list-item data-cy="rules-open" @click="shownDialog = 'rules'">
-          Rules
+          {{ `${t('game.menus.gameMenu.rules')}` }}
```

Comment:
```suggestion
          {{  t('game.menus.gameMenu.rules')  }}
```

I don't think we need the interpolation here or any of them in the template

Reply:
Correct we only need the ` ${} string template literals when we're combining multiple translations into a single string